src/shared/lib/HotspotSet.spec.ts

```diff
@@ -94,7 +106,10 @@ describe('HotspotSet', () => {
             [2, 6],
             [-5, 0],
         ]);
-        assert.deepEqual(hss._getHotspotRegions(), [[-5, 20], [24, 70]]);
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki